Source/Views/ActionSheetPrimaryView.swift

```diff
@@ -37,6 +37,7 @@ final class ActionSheetPrimaryView: UIView {
         self.configure(visualStyle: visualStyle)
 
         let backgroundBlur = self.buildBackgroundBlur(effect: visualStyle.blurEffect)
+        if visualStyle.backgroundColor != nil { backgroundBlur.effect = nil }
```

Comment:
Same comment as above, let's avoid calling `self.buildBackgroundBlur` altogether if a different background is provided through the visual style.

Reply:
I'm not sure that avoiding creating UIVisualEffect is the right call. I tried to make it as you said and the code became overloaded. It can be done for ActionSheetCancelActionView, but not here.